HTML/index.html

```diff
@@ -299,10 +302,15 @@ <h5 class="widget__title">Aspect Ratio</h5>
                         </aside><!-- /.sidebar -->
                         <table width=75% height=100px>
 							<tr>
-								<th>Laptop</th>
+								<th>Laptop </th>
 								<th>CPU</th>
 								<th>Price</th>
 							</tr>
+                            <tr>
```

Comment:
Remove this, this is not needed, PHP will create a table, we do not need to fill it with dummy data.